polaris.shopify.com/pages/_app.tsx

```diff
@@ -102,7 +103,12 @@ function MyApp({Component, pageProps}: AppProps) {
       </Head>
 
       <div
-        style={{background: isPolarisExample ? '#fafafa' : 'unset'}}
+        style={
+          {
+            background: isPolarisExample ? '#fafafa' : 'unset',
+            '--polaris-shopify-com-version': pkg.version,
```

Comment:
This will let us see which version is live in prod and let us better keep it up to date / make sure deploys are working